.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,69 @@
+﻿---
+name: "🐛 Bug Report"
+description: Create a report to help us improve
+labels: [bug]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: What is the problem? A clear and concise description of the bug.
+    validations:
+      required: true
+
+  - type: textarea
+    id: current
+    attributes:
+      label: Current behavior
+      description: |
+        Please include full errors, uncaught exceptions, screenshots, and relevant logs.
+        Using environment variable JFROG_CLI_LOG_LEVEL="DEBUG" upon running the command will provide more log information.
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction steps
+      description: |
+        Provide steps to reproduce the behavior.
+    validations:
+      required: false
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      description: |
+        What did you expect to happen?
+    validations:
+      required: false
+
+  - type: input
+    id: cli-security-version
+    attributes:
+      label: JFrog CLI-Security version
+    validations:
+      required: true
+      
+  - type: input
+    id: cli-version
+    attributes:
+      label: JFrog CLI version (if applicable)
+      description: using "jf --version"
+    validations:
+      required: false
```

```suggestion
  - type: input
    id: vs-extension-version
    attributes:
      label: JFrog Visual Studio Extension version
    validations:
      required: true
```
this needs to be changed to match the VS version. if we know this version we will know the CLI version and when we know that, we will see the CLI security version.